src/prefetch-links.ts

```diff
@@ -67,25 +67,40 @@ export class AddPrefetchLinks extends AsyncTransformStream<File, File> {
         console.warn(`Unable to get document ${documentUrl}: ${message}`);
         continue;
       }
-      const dependencies = [...document.getFeatures({
-                             kind: 'import',
-                             externalPackages: true,
-                             imported: true,
-                             noLazyImports: true
-                           })].filter((d) => !d.lazy);
+
+      const allDependencyUrls = [
+        ...document.getFeatures({
+          kind: 'import',
+          externalPackages: true,
+          imported: true,
+          noLazyImports: true
+        })
+      ].filter((d) => !d.lazy).map((d) => d.document.url.replace(/^\//, ''));
```

Comment:
I totes didn't need to strip these.  It was cargo'd paranoia from the earlier version of the code.  That code has now been removed.